bow-meta.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
You have excluded this file in `.gitignore` so maybe it would be interesting also if we remove from the repository.

Reply:
Added to gitignore and removed.